repl/src/main/scala/ammonite/repl/interp/Evaluator.scala

```diff
@@ -34,7 +34,9 @@ trait Evaluator{
   def processLine(code: String,
                   printCode: String,
                   printer: Printer,
-                  extraImports: Seq[ImportData] = Seq()): Res[Evaluated]
+                  extraImports: Seq[ImportData] = Seq(),
+                  exception_update:(Throwable) => Unit
```

Comment:
`onException` would probably be a better name.

Also, space after the `:` and no parens around the `Throwable`